symengine/derivative.cpp

```diff
@@ -410,8 +410,7 @@ static RCP<const Basic> diff(const CLASS &self, \
         return mul(polygamma(zero, arg), arg->diff(x));
     }
 
-    static RCP<const Basic> diff(const UnivariateIntPolynomial &self,
-            const RCP<const Symbol> &x) {
+    static RCP<const Basic> diff(const UnivariateIntPolynomial &self, const RCP<const Symbol> &x) {
```

Comment:
This one's not about the max number of characters on a line, but this is an unrelated change